docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -138,7 +138,7 @@ For further information about default suites, see ":ref:`Publishing and using Co
 Running a subset of queries in a CodeQL pack
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Additionally, you can include a path at the end of a pack specification to run a subset of queries inside the pack. This applies to any command that locates or runs queries within a pack.
+Additionally, CodeQL CLI v2.10.0 or later, you can include a path at the end of a pack specification to run a subset of queries inside the pack. This applies to any command that locates or runs queries within a pack.
```

Comment:
Is this correct? The [changelog entry](https://github.com/github/codeql-cli-binaries/blob/main/CHANGELOG.md#release-281-2022-02-15) suggests that this had been available since 2.8.1.

Reply:
You're right. Let me fix.

```suggestion
Additionally, CodeQL CLI v2.8.1 or later, you can include a path at the end of a pack specification to run a subset of queries inside the pack. This applies to any command that locates or runs queries within a pack.
```